pkg/clipboard/unclip.go

```diff
@@ -11,37 +12,43 @@ import (
 )
 
 // Clear will attempt to erase the clipboard.
-func Clear(ctx context.Context, checksum string, force bool) error {
-	if clipboard.Unsupported {
+func Clear(ctx context.Context, name string, checksum string, force bool) error {
```

Comment:
The nesting in this method is not very idiomatic. In Go you'd try to avoid else and else if constructs when possible.

Here you could just copy line 54/55 in the `clipboardClearCMD` branch and have it return there.

Then the unsupported case can just be a simple if and the rest doesn't need to be in an if statement at all.

Reply:
Resolved in a0c81ae6